nemo/collections/cv/modules/permute.py

```diff
@@ -0,0 +1,14 @@
+from nemo.core.classes import NeuralModule
+
+
+class Permute(NeuralModule):
```

Comment:
This looks generic and can probably be in some common folder

Reply:
moved to cv/parts/submodules